services/decoration.py

```diff
@@ -94,19 +92,12 @@ def determine_decoration_details(
 
         db_session = db_pull.get_db_session()
 
-        if org.service == "gitlab" and org.parent_service_id:
-            # need to get root group so we can check plan info
-            (gl_root_group,) = db_session.query(
-                func.public.get_gitlab_root_group(org.ownerid)
-            ).first()
+        # do not access plan directly - only through PlanService
+        org_plan = PlanService(current_org=org)
```

Comment:
The `PlanService` doesn't seem to go looking for the root org (looking at [main](https://github.com/codecov/shared/blob/9a61a94a70722971e56437defe5c31a2399928b0/shared/plan/service.py#L66))

I think I'm missing a code change... does this PR depend on an unmerged change to shared too?

Reply:
Ah yes, https://github.com/codecov/shared/pull/461 ok ok